RCAIDE/Library/Components/Thermal_Management/Batteries/Liquid_Cooled_Wavy_Channel.py

```diff
@@ -0,0 +1,95 @@
+           
+        
+        Source:
+           None
+        
+        """            
+         
+        self.tag                           = 'Wavy_Channel_Heat_Acquisition' 
+        self.heat_transfer_efficiency      = 1
+        self.coolant                       = Glycol_Water()
+        self.coolant_Reynolds_number       = 1.
+        self.coolant_velocity              = 1.
+        self.coolant_flow_rate             = 1
+        self.coolant_inlet_temperature     = None
+        self.coolant_hydraulic_diameter    = 1.
+        self.channel_side_thickness        = 0.001                  # Thickness of the Chanel through which condcution occurs 
+        self.channel_top_thickness         = 0.001                  # Thickness of Channel on the top where no conduction occurs
+        self.channel_width                 = 0.005                  # width of the channel 
+        self.channel_height                = 0.003                  # height of the channel 
+        self.channel_contact_angle         = 47.5 * Units.degrees   # Contact Arc angle in degrees    
+        self.channel                       = Aluminum()
+        self.channel_aspect_ratio          = 1. 
+        self.channels_per_module           = 1
+        self.battery_contact_area          = 1.
+        self.contact_area_per_module       = 1.  
+        self.power_draw                    = 1. 
+        self.single_side_contact           = True 
+        self.design_heat_removed           = None   
+        self.percent_operation             = 1.0
+        self.type                          = 'Liquid'
+        
+        return
+    
+    def __init__ (self, distributor=None):
+        
+        """This creates Reservoir and Heat Exchanger containers if it does not exist on
+            the coolant line as a liquid cooled system requires these and cannot operate without.
+    
+        Assumptions:
+            None
+        
+        Source:
+            None
+        """               
+        distributor.reservoirs                     = Container()
+        distributor.heat_exchangers                = Container()
+        
+    def append_operating_conditions(self,segment,coolant_line,add_additional_network_equation = False):
+        append_wavy_channel_conditions(self,segment,coolant_line,add_additional_network_equation)
+        return
+    
+    def append_segment_conditions(self, segment,coolant_line, conditions):
+        append_wavy_channel_segment_conditions(self, segment,coolant_line, conditions)
+        return
+    
+    def compute_thermal_performance(self,battery,coolant_line, Q_heat_gen,T_cell,state,dt,i):
+        T_battery_current =  wavy_channel_rating_model(self, battery,coolant_line, Q_heat_gen, T_cell, state, dt, i)
+        return  T_battery_current
+    
+    def plot_operating_conditions(self, results, coolant_line,save_filename,save_figure = False,show_legend = True,file_type = ".png",
```

Comment:
#See previous comment. we also need to be consistent, dt, or delta_t throughout the code. Also also, t_idx or i or t_i - pick one

Reply:
Everything is now delta_t and t_idx